app/views/includes/footer.nunjucks.old

```diff
@@ -0,0 +1,17 @@
+<div class="global-footer">
+  <div class="global-footer__inner">
+
+    <div>
+      <h2 class="util-visuallyhidden">Support links</h2>
+      <ul class="link-list">
+        <li>
+          <a href="https://beta.nhs.uk/help/clinical-assurance">Clinical assurance</a>
+        </li>
+        <li>
+          <a href="https://beta.nhs.uk/help/cookies">Cookies</a>
+        </li>
+      </ul>
+    </div>
+
```

Comment:
Can this blank line be deleted if it isn't required?

Reply:
File shouldn't be there, removed...